### DISCUSSION TITLE
L'automazione non parte

### DISCUSSION THREAD

**Discussion by StefanoVitale69** [indomus/forum#2161] [HUB: Home Assistant]
Salve, ho creato un'automazione che, quando il livello della batteria del cellulare, su cui ho l'app Home Assistant con il relativo sensore attivato, scende sotto un certa soglia, dovrebbe far partire un messaggio.
Purtroppo l'automazione non parte; ho l'impressione che il valore trasmesso dal sensor dell'app Home Assistant sia un valore stringa, e quindi il trigger non si attivi 
Come posso risolvere ?

```
- id: '1670498405326'
  alias: Messaggio batteria scarica Note
  description: ''
  trigger:
  - platform: numeric_state
    entity_id: sensor.samsung_note20_battery_level
    below: 95
  condition: []
  action:
  - service: tts.google_cloud_say
    data:
      entity_id: media_player.gruppo_casa
      message: Stefano, il livello della batteria del telefono è sceso al     {{states.sensor.samsung_note20_battery_level.state}}
        %. Ti consiglio di mettere a caricare il telefono
  mode: single

**Comment by snakuzzo** [comment]
Non credo sia quello il problema.
Ricordati che il trigger scatta solo quando la batteria passa da un valore SUPERIORE O UGUALE a 95 ad un valore INFERIORE a 95.
Se è già ad un valore sotto 95 non può scattare. Per scattare deve prima tornare sopra i 95 e poi sotto
